ui/app/i18n/home/locale_en.json

```diff
@@ -46,6 +46,9 @@
     "COULD_NOT_GET_PROVIDER": "Could not get provider for the current user.",
     "WARNING_SERVER_TIME_ZONE_MISMATCH": "The server timezone ({{serverTime}}) is different from your browser's time/timezone ({{localeTimeZone}}). This can cause some data or reports to appear incorrect. Please contact the administrator for further assistance.",
     "LOGIN_PAGE_HEADER_TEXT": "BAHMNI EMR LOGIN",
+    "LOGIN_PAGE_HEADER_TEXT_1": "REPUBLIC OF MOZAMBIQUE",
+	"LOGIN_PAGE_HEADER_TEXT_2": "MINISTRY OF HEALTH",
+	"LOGIN_PAGE_HEADER_TEXT_3": "ELECTRONIC PATIENT TRACKING SYSTEM (EPTS)",
```

Comment:
Can you remove the extra spaces?

Reply:
@krishnanspace , I will remove the extra spaces. By reducing the height by 25%, it also reduced the width automatically to keep the proportions